dlc-wallet/src/main/scala/org/bitcoins/dlc/wallet/DLCWallet.scala

```diff
@@ -43,6 +44,9 @@ abstract class DLCWallet extends Wallet with AnyDLCHDWalletApi {
   private[bitcoins] val dlcSigsDAO: DLCCETSignatureDAO = DLCCETSignatureDAO()
   private[bitcoins] val dlcRefundSigDAO: DLCRefundSigDAO = DLCRefundSigDAO()
 
+  private[bitcoins] val txDAO: TransactionDAO = TransactionDAO()
```

Comment:
You don't need to do this here, `Wallet` already has it, you can use `transactionDAO`